app/controllers/school_groups_controller.rb

```diff
@@ -90,7 +90,17 @@ def current_dashboard
 
   def enhanced_dashboard
     if can?(:compare, @school_group)
-      render 'recent_usage'
+      respond_to do |format|
+        format.html do
+          render 'recent_usage'
+        end
+        format.csv do
+          metric = params['metric'] || 'change'
+          metric_label = I18n.t("school_groups.show.metric.#{metric}")
+          send_data SchoolGroups::RecentUsageCsvGenerator.new(school_group: @school_group, metric: metric).export,
+          filename: "#{@school_group.name} - #{I18n.t('school_groups.titles.recent_usage')} - #{metric_label}.csv"
```

Comment:
Can you add the date to the filename? So if someone is regularly requesting this, then the versions are clear